debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/valueholder/LmDDLLCR.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.oracle.logminer.valueholder;
+
+/**
+ * This class mimics some API of oracle.streams.DDLLCR interface
+ *
+ */
+public interface LmDDLLCR extends LmLCR {
```

Comment:
Should be named `LmDdlLcr` and `LmLcr` (I see now that I hadn't done this myself correctly before; in general, abbreviations in type names should be given normal camel-casing).